static/css/styles.css

```diff
@@ -370,3 +370,10 @@ H4 { margin-bottom: 0.21cm }
 H4.western { so-language: en-GB }
 H4.cjk { font-family: "SimSun" }
 H4.ctl { font-family: "Arial" }
+  
+li

```

Comment:
This is a global style to all `<li>` elements. It should be avoided, since many pages use the `<li>` element.
If you're trying to target some specific `<li>` elements, then target by a class for them please.
This will not be accepted since MANY things use the `<li>` element.

Reply:
Understood - please see below